utils/update-db.ts

```diff
@@ -350,11 +372,36 @@ class KanjiDictParser extends Writable {
   }
 }
 
+interface HeisigData {
```

Comment:
Using (H/h)eisigData to refer to all of the data and the format for a single kanji's data is a little confusing.

Reply:
Made singular type HeisigDatum